### PR TITLE
updated guide-connect-supabase.md

### PR DESCRIPTION
**Pull request by TomasBankauskas**

### CODE REVIEW

File: src/content/config.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { defineCollection, z } from 'astro:content';
 
 const guides = defineCollection({
     schema: z.object({
-        title: z.string(),
+        title: z.string().optional(),
```
Comment (aitchiss) [inline src/content/config.ts:5]: Do we prefer to make this optional, or should we just make "Set up Supabase database" as the title for the page?

File: src/content/guides/guide-connect-supabase.md [hunk 2]
```diff
@@ -50,13 +44,9 @@ Svelte,https://svelte.dev/,svelte.svg,0,"Svelte is a UI framework that uses a co
...
 
-## Install the Supabase Netlify extension
-
-Now we can install the [Supabase extension](https://app.netlify.com/extensions/supabase). In the Netlify UI, go to your team’s dashboard, navigate to **Extensions** and click on the Supabase extension. Click the install button to install the extension.
-
-### Configure the Supabase extension
+## Configure the Supabase Netlify extension
 
-After the extension is installed, navigate to the Supabase template site that you deployed, and go to **Site configuration**. In the **General** settings, you will see a new **Supabase** section. Click **Connect** to connect your Netlify site to your Supabase account using OAuth.
+The [Netlify Supabase extension](https://app.netlify.com/extensions/supabase) should already be installed. Visit your site's configuration page and scroll to the Supabase section. Click **Connect** to connect your Netlify site to your Supabase account using OAuth.
```
Comment (aitchiss) [inline src/content/guides/guide-connect-supabase.md:49]:
```suggestion
The [Supabase Netlify extension](https://app.netlify.com/extensions/supabase) should already be installed. Visit your site's configuration page and scroll to the Supabase section. Click **Connect** to connect your Netlify site to your Supabase account using OAuth.
```
Sorry this was my copy error - let's match the title as I think that's the correct name

File: src/content/config.ts (hunk 1 above)
Comment (TomasBankauskas) [reply src/content/config.ts:5]: Yep, lets use "Set up Supabase database" for the title

Comment (aitchiss) [reply src/content/config.ts:5]: sounds good - I think we can undo this change then 👍

**Pull request merged**